src/components/common/transmission/transmission.html

```diff
@@ -9,7 +9,8 @@
         <ion-row class="spacing-row"></ion-row>
         <ion-row [formGroup]="formGroup">
             <ion-col size="48">
-                <input type="radio" [id]="'wrtc-transmission-manual-'+uniqueId"
+                <input type="radio" [id]="'transmission-manual-'+uniqueId"
+                       id="transmission-manual"
```

Comment:
there are now 2 id attributes on this element

Reply:
good spot. i looked to replace the existing one as couldnt see the reason for it.  Queried cai and restored the original but without the wrtc as this is a shared component used on multiple pages. will remove the second one